client/src/containers/CampaignAdmin/CampaignAdminItem.js

```diff
@@ -121,20 +124,20 @@ class CampaignAdminItem extends Component {
     if (campaign.State === 'open') {
       switch (item.ChangeType) {
         case 'created':
-          badge.className = 'badge badge-warning list-group-item__status';
+          badge.status = 'draft';
           badge.Title = i18n._t('CampaignAdmin.DRAFT', 'Draft');
           break;
         case 'modified':
-          badge.className = 'badge badge-warning list-group-item__status';
+          badge.status = 'modified';
           badge.Title = i18n._t('CampaignAdmin.MODIFIED', 'Modified');
           break;
         case 'deleted':
-          badge.className = 'badge badge-error list-group-item__status';
+          badge.status = 'removed';
           badge.Title = i18n._t('CampaignAdmin.REMOVED', 'Removed');
```

Comment:
I couldn't see a way of reproducing this via the UI. When I remove a page from a campaign is just removes it from the items list.